packages/arcgis-rest-auth/src/UserSession.ts

```diff
@@ -103,7 +103,7 @@ export interface IOAuth2Options {
   provider?: AuthenticationProvider;
 
   /**
-   * The requested validity in minutes for a token. Defaults to 20160 (two weeks).
+   * Duration (in minutes) that a token will be valid for browser-based OAuth 2.0 or the refresh token will be valid for server-based OAuth 2.0. The default is 20160 (two weeks) for both.
```

Comment:
```suggestion
   * The requested validity in minutes of `access_token` for implicit grants or `refresh_token` for explicit grants (authorization grants).
```

Should we change the wording to this (closer to the language on [this page](https://developers.arcgis.com/rest/users-groups-and-items/authorize.htm))?